finetune/base.py

```diff
@@ -156,14 +157,18 @@ def finetune(self, Xs, Y=None, batch_size=None):
         batch_size = batch_size or self.config.batch_size
 
         val_input_fn, train_input_fn, val_size, val_interval = self.input_pipeline.get_train_input_fns(Xs, Y, batch_size=batch_size)
-        if val_size <= 10 and self.config.keep_best_model:
-            tf.logging.warning(
-                "Early stopping / keeping best model with a validation size of {} is likely to case undesired results".format(val_size))
+        if self.config.keep_best_model:
+            if isinstance(val_size, dict):
+                tf.logging.warning("Cannot early stop or keep best model with MTL")
```

Comment:
We shouldn't change this in this PR but we should consider thinking about how early stopping should work for an MTL problem.  I.e. mean loss across tasks or something stops decreasing.